taxi/views.py

```diff
@@ -1,3 +1,16 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Car, Driver, Manufacturer
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    drivers = Driver.objects.count()
+    manufacturers = Manufacturer.objects.count()
+    cars = Car.objects.count()
+    context = {
+        "drivers": drivers,
+        "manufacturers": manufacturers,
+        "cars": cars,
```

Comment:
change "drivers" to num_drivers, "manufacturers" to num_manufacturers, "cars" to num_cars. This fix test error

Reply:
tysm!